.github/workflows/eya-def-tools-python-package.yml

```diff
@@ -17,27 +17,31 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python ${{matrix.python-version}}
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: ${{matrix.python-version}}
+    - name: Update apt-get
+      run: sudo apt-get update
+    - name: Install graphviz
+      run: sudo apt-get install graphviz graphviz-dev
```

Comment:
This allows installing the `pygraphviz` package and run the tests of diagram rendering in the GitHub Actions pipeline, which were previously skipped.